### ISSUE TITLE
`ClassMemberImpliedModifier` xdoc examples: `<property>` not nested inside self-closed module tag

### ISSUE THREAD

**Issue by smita1078** [checkstyle/checkstyle#20795]
`ClassMemberImpliedModifier` xdoc examples: `<property>` not nested inside self-closed module tag

In `src/xdocs-examples/resources/com/puppycrawl/tools/checkstyle/checks/modifier/classmemberimpliedmodifier/`, Example2, Example3, and Example4 declare their config property outside the module element due to a self-closing tag:

```xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="ClassMemberImpliedModifier" />
      <property name="violateImpliedStaticOnNestedEnum" value="false"/>
  </module>
</module>
```

Because `<module name="ClassMemberImpliedModifier" />` is self-closed, the following `<property>` element becomes a child of `TreeWalker`, not of `ClassMemberImpliedModifier`. This means:

1. `XdocsExamplesAstConsistencyTest#testEveryPropertyHasAnExample` fails, since none of the three properties (`violateImpliedStaticOnNestedEnum`, `violateImpliedStaticOnNestedInterface`, `violateImpliedStaticOnNestedRecord`) are actually configured on the check in any example.
2. More importantly, these examples likely never actually ran with the intended property set to `false` — meaning the `// violation` comments in the example code may have been written/verified against the *default* configuration rather than the documented one.

**Fix:** properly nest the `<property>` inside the module element, e.g.:

```xml
<module name="Checker">
  <module name="TreeWalker">
    <module name="ClassMemberImpliedModifier">
      <property name="violateImpliedStaticOnNestedEnum" value="false"/>
    </module>
  </module>
</module>
```
